test/Atc.Rest.FluentAssertions.Tests/Assertions/CreatedResultAssertionsTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Atc.Rest.FluentAssertions.Tests.Assertions
+{
+    public class CreatedResultAssertionsTests : ContentResultAssertionsBaseFixture
+    {
+        [Fact]
+        public void Ctor_Sets_Subject_On_Subject_Property()
+        {
+            // Arrange
+            var expected = new ContentResult();
+
+            // Act
+            var sut = new CreatedResultAssertions(expected);
+
+            // Assert
```

Comment:
// Act & Assert

Reply:
Dont agree. The test "Ctor sets subject on subject property", i.e. the "act" part of the test is new'ing up SUT.